lib/timers.js

```diff
@@ -215,6 +215,16 @@ function setInterval(callback, repeat, arg1, arg2, arg3) {
   return timeout;
 }
 
+
+ObjectDefineProperty(setInterval, customPromisify, {
+  enumerable: true,
+  get() {
+    if (!timersPromises)
+      timersPromises = require('timers/promises');
```

Comment:
This would be a good candidate for null coalescing assignment if we don't plan to backport to Node.js v14.x.
```suggestion
    timersPromises ??= require('timers/promises');
```

Reply:
removed